### PR TITLE
feat: _Quality gate fixes_

### LINKED ISSUE #64: chore: use new helm lint and test workflow
Please use the new [helm lint and test workflow from this TRG](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-09) and adjust it to your needs.

### PR DESCRIPTION
**Pull request by adkumar1**
First merge PR #68 
- This PR consist for fixes for QG4
- Fixes #64

### CODE REVIEW

File: charts/sdfactory/README.md [hunk 1]
```diff
@@ -19,23 +19,26 @@ Helm Charts for SD Factory application. Self-Description Factory component is re
 | autoscaling.targetCPUUtilizationPercentage | int | `80` |  |
 | fullnameOverride | string | `""` |  |
 | image.pullPolicy | string | `"Always"` | Set the Image Pull Policy |
-| image.repository | string | `"ghcr.io/catenax-ng/tx-sd-factory/sdfactory"` | Image to use for deploying an application |
-| image.tag | string | `""` | Image tage is defined in chart appVersion. |
+| image.repository | string | `"tractusx/sdfactory"` | Image to use for deploying an application |
+| image.tag | string | `"main"` | Image tage is defined in chart appVersion. |
```
Comment (almadigabor) [inline charts/sdfactory/README.md:23]: Do not use main as the default tag for the image. Leave it blank and set it in the template to the Chart.yaml appVersion

File: charts/sdfactory/values.yaml [hunk 2]
```diff
@@ -32,7 +32,7 @@ image:
     # -- Set the Image Pull Policy
   pullPolicy: Always
     # -- Image tage is defined in chart appVersion.
-  tag: ""
+  tag: "main"
```
Comment (almadigabor) [inline charts/sdfactory/values.yaml:35]: Do not use main as the default tag for the image. Leave it blank and set it in the template to the Chart.yaml appVersion

**Review by almadigabor — CHANGES_REQUESTED** [review]
Added comments.

**Comment by almadigabor** [comment]
There are also issues with the url in the upgrade workflow: https://github.com/eclipse-tractusx/sd-factory/actions/runs/5922843209/job/16057431576?pr=69#step:5:7

**Comment by almadigabor** [comment]
And please solve merge conflicts.

**Comment by adkumar1** [comment]
resolved

**Pull request merged**